dts/msc-sm2-imx6x.dtsi

```diff
@@ -648,6 +648,11 @@
 	clock-frequency = <100000>;
 	status = "okay";
 
+	carrierdata@57 {
+		compatible = "atmel,24c32";
+		reg = <0x57>;
+		pagesize = <32>;
```

Comment:
I think we should make this read-only by adding this property.

Reply:
Good point.

Reply:
No, we can't: This is our carrier board data EEPROM.